brainkeeper/src/app/classes/person.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * A person that the player must guess correctly during the game.
+ */
+export class Person {
+    private _id: number | null;
+    private _name: string;
+    private _picture: string;
+
+    constructor(name: string, picture: string, id?: number) {
+        if (!name) {
+            throw new Error('Name must not be null/empty/undefined');
+        }
+        if (!picture) {
+            throw new Error('Picture must not be null/empty/undefined');
+        }
+        if (id !== undefined) {
+            this.checkId(id);
+        }
+        this._id = id === undefined || id == null ? null : id;
+        this._picture = picture;
+        this._name = name;
+    }
+
+
+    /**
+     * Sets the person's id (must be a positive finite integer).
+     */
+    public set id(id: number) {
+        this.checkId(id);
+        this._id = id;
+    }
+
+    /**
+     * The person's id or null when no id is assigned.
+     */
+    public get id(): number | null {
+        return this._id == null ? null : this._id;
```

Comment:
Is this conditional return value really necessary?

Reply:
Fixed with 717f448